Spot markets (#1024)
vegaprotocol/documentation: docs/concepts/liquidity/provision.md

The bond committed during the liquidity commitment transaction is stored in a bond account (one per party, per market). Deposits and withdrawals for the bond account are governed by the protocol and cannot be manually triggered.

### Liquidity commitment for a spot market
In almost all ways, providing liquidity to a spot market is the same as for other markets. There are minor differences when considering the commitment and how much to have available to support a spot market commitment. 

Providing liquidity to a spot market requires setting aside a set amount of commitment as bond, using the market's quote asset. The orders for the commitment need to provide the full amount of assets required to support the liquidity for both the buy and sell side.

#### Example

> An LP commits 100,000 USDT to the bond account for a BTC/USDT market.
> 
> That LP has committed to providing 100,000 USDT worth of buy orders and sell orders.

Reviewer: Small nitpick as I think worth avoiding the assumption that a commitment amount of 100,000 USDT results in requiring 100,000 USDT worth of orders. This won't always be the case due to the bond scaling.

Maybe just...
```suggestion
An LP has submitted a commitment and has an obligation to provide 100,000 USDT worth of buy orders and sell orders.
```
Want to avoid
